### PR TITLE
SCANNERAPI-178 Increase timeout for downloading bootstrap jar

### PR DESCRIPTION
**Pull request by ehartmann**

### CODE REVIEW

File: api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java [hunk 1]
```diff
@@ -76,13 +74,6 @@ public void task_should_not_require_project() {
     assertThat(Utils.taskRequiresProject(props)).isFalse();
   }
 
-  @Test
-  public void delete_quietly() {
-    Path f = mock(Path.class);
-    doThrow(IOException.class).when(f).getFileSystem();
-    Utils.deleteQuietly(f);
```
Comment (henryju) [inline api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java]: Why have you removed this test?

Comment (ehartmann) [reply api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java]: I've upgraded all dependencies to be up to date. And now Mockito is checking that the signature of the method is allowing to throw the kind of exception we ask it to throws. 

In this case `getFileSystem()` cannot throw an `IOException`, we would like that `Files#walkFileTree` to throw an `IOException` to check that `deleteQuietly` is resilient to such case.
However we have to use PowerMockito to mock static methods, I though it was a little too much to include PowerMockito to test this. 

Obviously, if you think that this test must remain, I can either add PowerMockito or downgrading Mockito, as you likes.

Comment (henryju) [reply api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java]: Can't you throw another exception instead? Any runtime exception would fit. Here I think the purpose is to be sure deleteQuietly will silently ignore all exceptions.

Comment (ehartmann) [reply api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java]: I've tried that, but `deleteQuietly` is only catching `IOException`, so in this case the test is failing with a RuntimeException not catched.  :(

Comment (henryju) [reply api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java]: ok, then remove the test, I have no better idea...

File: api/src/main/java/org/sonarsource/scanner/api/internal/OkHttpClientFactory.java [hunk 2]
```diff
@@ -135,6 +138,7 @@ private static SSLSocketFactory systemDefaultSslSocketFactory(X509TrustManager t
 
     String defaultKeyStorePassword = System.getProperty("javax.net.ssl.keyStorePassword", "");
     char[] passwd = defaultKeyStorePassword.isEmpty() ? null : defaultKeyStorePassword.toCharArray();
+    logger.debug("keyStore password : " + passwd);
```
Comment (henryju) [inline api/src/main/java/org/sonarsource/scanner/api/internal/OkHttpClientFactory.java]: Are you sure we should keep that?

**Pull request merged**